extensions/wikia/CoppaTool/CoppaToolController.class.php

```diff
@@ -152,9 +152,9 @@ public function renameIPAddress() {
 			'notify_renamed' => false,
 			'reason' => wfMessage( 'coppatool-reason' )->plain(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change the output format of this i18n string away from plain(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)

[project-id]: wikia:app